scripts/docker-tests-runner.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+yarn build-tests
+
+npx ts-node -- \
```

Comment:
Need to create another script that will run these tests against tests served by `webpack-dev-server`.

Also `webpack.tests.config` are missing `RenderingTestResultServer` middleware.

Reply:
I think done.